server/routes/scene.routes.js

```diff
@@ -0,0 +1,38 @@
+const sceneRouter = require('express').Router();
+const Scene = require('../models/Scene.model.js');
+
+function addAScene(request, response, next) {
+  console.log('Incoming', request.body);
+
+  if(!request.body) {
+    let err = new Error('You must provide a scene');
+    err.status = 400;
+    next(err);
+    return;
+  }
+
+  let theSceneCreated = new Scene({
+    sceneImage: request.body.sceneImage,
+    sceneText: request.body.sceneText,
+    sceneChoice: {
+      choiceText: request.body.sceneChoice.choiceText,
+      choiceScore: request.body.sceneChoice.choiceScore
+    }
+  });
+  console.log(theSceneCreated);
```

Comment:
remove unneeded console log statements, or leave them since I need to validate when I code the gameplay page?